### PR TITLE
Merchant client is unusable

### PR DESCRIPTION
**Pull request by thxmasj**
It was never possible to add a base URL that worked with both the authorization server (access-token-api) and the resource server (payment-api/payment-token-api). Changed the configuration of the client to take in the authorization server URL explicitly instead. Also removed the version (v1) from the base URL, as it is not specific to environment. Did some small changes to operation ids as well to improve consistency a bit. Finally, added integration tests including `t1` and `test` to make sure the clients work with those environments now. Didn't take the effort to create full working requests, but just validating that requests go through and fail with 404 for now.

### CODE REVIEW

File: java/webflux-client/src/main/java/no/bankaxept/epayment/client/webflux/WebFluxClient.java [hunk 1]
```diff
@@ -18,7 +18,7 @@ public class WebFluxClient implements HttpClient {
   private final WebClient webClient;
 
   public WebFluxClient(String baseUrl) {
-    webClient = WebClient.create(baseUrl);
+    webClient = WebClient.create(baseUrl.endsWith("/") ? baseUrl : baseUrl + "/");
```
Comment (k-irishina) [inline java/webflux-client/src/main/java/no/bankaxept/epayment/client/webflux/WebFluxClient.java:21]: Why was this change needed?

Comment (k-irishina) [reply java/webflux-client/src/main/java/no/bankaxept/epayment/client/webflux/WebFluxClient.java:21]: Wouldn't it be more common to prefix the uri paths with a slash instead? I think the underlying UriBuilder would handle any excessive ones, perhaps?

File: java/merchant-client/src/main/java/no/bankaxept/epayment/client/merchant/MerchantClient.java [hunk 2]
```diff
@@ -41,8 +32,23 @@ public MerchantClient(BaseClient baseClient) {
     this.baseClient = baseClient;
   }
 
-  public MerchantClient(String baseurl, String apimKey, String username, String password) {
-    this.baseClient = new BaseClient.Builder(baseurl).apimKey(apimKey).withScheduledToken(username, password).build();
+  public MerchantClient(
+      URL authorizationServerUrl,
```
Comment (k-irishina) [inline java/merchant-client/src/main/java/no/bankaxept/epayment/client/merchant/MerchantClient.java:36]: Is taking in a URL instead of a string supposed to prevent malformed URLs being passed in?

File: java/webflux-client/src/main/java/no/bankaxept/epayment/client/webflux/WebFluxClient.java (hunk 1 above)
Comment (thxmasj) [reply java/webflux-client/src/main/java/no/bankaxept/epayment/client/webflux/WebFluxClient.java:21]: Nope, it doesn't. And that's the reason. :-)

File: java/merchant-client/src/main/java/no/bankaxept/epayment/client/merchant/MerchantClient.java (hunk 2 above)
Comment (thxmasj) [reply java/merchant-client/src/main/java/no/bankaxept/epayment/client/merchant/MerchantClient.java:36]: Yes

File: java/webflux-client/src/main/java/no/bankaxept/epayment/client/webflux/WebFluxClient.java (hunk 1 above)
Comment (thxmasj) [reply java/webflux-client/src/main/java/no/bankaxept/epayment/client/webflux/WebFluxClient.java]: Scratch that, you're right. Fixed.

**Review by k-irishina — APPROVED** [review]
⚡
